app/controllers/webhooks_controller.rb

```diff
@@ -193,14 +212,16 @@ def handle_payment_intent_succeeded(
       # Send separate email for each seller.
       grouped_items = items.group_by { |li| li['seller_id'] }
       grouped_items.each_key do |sid|
-        EmailManager::DonationReceiptSender.call(
-          {
-            payment_intent: payment_intent,
-            amount: grouped_items[sid].map { |li| li['amount'].to_f }.sum,
-            merchant: Seller.find_by(seller_id: sid).name,
-            email: payment_intent.purchaser.email
-          }
-        )
+        if sid.present?
```

Comment:
This means that we don't send emails for light up chinatown donatoins?

Reply:
`seller_id` can be null now, when there is a `project_id`, so we should skip it here (the grouping logic seemed weird, and it should be on it's own anyway). I'm triggering a separate email in the case above for a project